packages/Search/src/details/DTK_DetailsUtils.hpp

```diff
@@ -18,62 +18,92 @@
 namespace DataTransferKit
 {
 
-/** \brief Assigns the given value to all elements in the view.
- */
 template <typename T, typename DeviceType>
-void fill( Kokkos::View<T *, DeviceType> out, T const &value )
+class ExclusiveScanFunctor
```

Comment:
Add a comment explaining why you are using a functor instead of a lambda function